app/models/user.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable require-atomic-updates */
+const bcrypt = require('bcrypt');
+
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define(
+    'users',
```

Comment:
for convention, model's name must be singular.

Reply:
Changed